ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-maistra-2.4__servicemesh-ocp4.13-lp-interop.yaml

```
@@ -0,0 +1,67 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  kiali-cypress-tests-runner:
+    name: kiali-cypress-tests
+    namespace: ci
+    tag: v1.57
+  kiali-int-tests-runner:
+    name: kiali-int-tests
+    namespace: ci
+    tag: v1.57
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: maistra-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: servicemesh-interop-aws
+  cron: 0 1 * * 1
```

```suggestion
  cron: 0 6 * * 1
```
These cron jobs use UTC. This should set the trigger time correctly